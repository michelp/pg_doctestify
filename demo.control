comment = 'Demo extension'
default_version = '1.0'
relocatable = false
module_pathname = ''
