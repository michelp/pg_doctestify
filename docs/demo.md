# pg_doctestify Demo

This is a simple test, It doesn't do anything special.
``` postgres-console
select version();
                                                       version                                                       
---------------------------------------------------------------------------------------------------------------------
 PostgreSQL 17.5 (Debian 17.5-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
(1 row)
```
## Another simple thing
``` postgres-console
select * from generate_series(1, 10);
 generate_series 
-----------------
               1
               2
               3
               4
               5
               6
               7
               8
               9
              10
(10 rows)
```

The source for this test is [sql/demo.sql](https://github.com/michelp/pg_doctestify/blob/main/sql/demo.sql).
The rendered markdown is [docs/demo.md](https://github.com/michelp/pg_doctestify/blob/main/docs/demo.md)