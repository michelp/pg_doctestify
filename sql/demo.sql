-- # This is a demo test.
--
-- It doesn't do anything special.
select version();
