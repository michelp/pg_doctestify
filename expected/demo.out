-- # This is a demo test.
--
-- It doesn't do anything special.
select version();
                                                       version                                                       
---------------------------------------------------------------------------------------------------------------------
 PostgreSQL 17.5 (Debian 17.5-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
(1 row)

